api/prisma/migrations/01_change_to_prisma/migration.sql

```diff
@@ -1,997 +1,667 @@
-
-
-ALTER TABLE "household_member"
-DROP COLUMN "work_in_region";
-
-
-ALTER TABLE "household_member" ADD COLUMN "work_in_region" "yes_no_enum";
-
-
-UPDATE "household_member"
-SET "work_in_region" = CAST("work_in_region_TEMP" AS "yes_no_enum");
-
-
-ALTER TABLE "household_member"
-DROP COLUMN "work_in_region_TEMP";
-
--- AlterTable "jurisdictions"
-
+-- AlterTable
 ALTER TABLE "jurisdictions" RENAME CONSTRAINT "PK_7cc0bed21c9e2b32866c1109ec5" TO "jurisdictions_pkey";
-
-
-ALTER TABLE "jurisdictions"
-ALTER COLUMN "updated_at"
-DROP DEFAULT;
-
-
-ALTER TABLE "jurisdictions" ADD COLUMN "listing_approval_permissions_TEMP" "jurisdictions_listing_approval_permissions_enum" [];
-
-
-UPDATE "jurisdictions"
-SET "listing_approval_permissions_TEMP" = "listing_approval_permissions";
-
-
-ALTER TABLE "jurisdictions"
-DROP COLUMN "listing_approval_permissions";
-
-
-ALTER TABLE "jurisdictions" ADD COLUMN "listing_approval_permission" "user_role_enum" [];
-
+ALTER TABLE "jurisdictions" ADD COLUMN     "enable_accessibility_features" BOOLEAN NOT NULL DEFAULT false;
+ALTER TABLE "jurisdictions" ADD COLUMN     "enable_geocoding_preferences" BOOLEAN NOT NULL DEFAULT false;
+ALTER TABLE "jurisdictions" ADD COLUMN     "enable_partner_settings" BOOLEAN NOT NULL DEFAULT false;
+ALTER TABLE "jurisdictions" ADD COLUMN     "enable_utilities_included" BOOLEAN NOT NULL DEFAULT false;
+ALTER TABLE "jurisdictions" ADD COLUMN     "listing_approval_permission" "user_role_enum"[];
+ALTER TABLE "jurisdictions" ADD COLUMN     "rental_assistance_default" TEXT;
+ALTER TABLE "jurisdictions" ALTER COLUMN "updated_at" DROP DEFAULT;
 
 UPDATE "jurisdictions"
-SET "listing_approval_permission" = CAST(CAST("listing_approval_permissions_TEMP" AS TEXT []) AS "user_role_enum" []);
-
-
-ALTER TABLE "jurisdictions"
-DROP COLUMN "listing_approval_permissions_TEMP";
+SET "rental_assistance_default" = 'Housing Choice Vouchers, Section 8 and other valid rental assistance programs will be considered for this property. In the case of a valid rental subsidy, the required minimum income will be based on the portion of the rent that the tenant pays after use of the subsidy.';
 
+ALTER TABLE "jurisdictions" ALTER COLUMN "rental_assistance_default" SET NOT NULL;
 
 ALTER TABLE "jurisdictions" ADD COLUMN "languages_TEMP" "jurisdictions_languages_enum" [];
 
-
 UPDATE "jurisdictions"
 SET "languages_TEMP" = "languages";
 
-
 ALTER TABLE "jurisdictions"
 DROP COLUMN "languages";
 
-
 ALTER TABLE "jurisdictions" ADD COLUMN "languages" "languages_enum" [] DEFAULT ARRAY ['en'] :: "languages_enum" [];
 
-
 UPDATE "jurisdictions"
 SET "languages" = CAST(CAST("languages_TEMP" AS TEXT []) AS "languages_enum" []);
 
-
 ALTER TABLE "jurisdictions"
 DROP COLUMN "languages_TEMP";
 
 -- AlterTable
-
 ALTER TABLE "listing_events" RENAME CONSTRAINT "PK_a9a209828028e14e2caf8def25c" TO "listing_events_pkey";
+ALTER TABLE "listing_events" ADD COLUMN     "start_date" TIMESTAMPTZ(6);
```

Comment:
issue: This column is added, but the data is not getting populate with the data transfer. And the UI is looking at the start_date to determine if we show the lottery date

This PR for one property
![image](https://github.com/user-attachments/assets/17014734-5aa0-4587-bc91-31e80aa35f9c)

Same in detroit dev
![image](https://github.com/user-attachments/assets/bb55d99d-65c3-48de-a30e-5d7f63ef6599)